arrocke/gloss-translation
# feat: reorder translations

## What has changed

- Added `ReorderableMultiselectInput`
- Used `ReorderableMultiselectInput` for translation selector on the manage language page

## Connected Issues


Closes #208

## QA Steps


1. Go to the manage language page for English
2. Add "World English Bible"
3. Add "Wycliffe's Bible with Modern Spelling (Enhanced)"
4. Save changes
5. Go to Malachi 4:6
    - The translation should use World English Bible
6. Go to Matthew 1:1
    - The translation should use World English Bible
1. Go to the manage language page for English
2. Move "Wycliffe's Bible with Modern Spelling (Enhanced)" above "World English Bible"
3. Save changes
4. Go to Malachi 4:6
    - The translation should use Wycliffe's Bible
5. Go to Matthew 1:1
    - The translation should use World English Bible

## Post-Deployment
